SPECS-EXTENDED/python-breathe/python-breathe.signatures.json

```diff
@@ -0,0 +1,9 @@
+{
+ "Signatures": {
+  "breathe-4.35.0.tar.gz": "55b54723752fc04b892a0f868782b1df65e69db6ca94fb32cf04be495bfd7841",
+  "956.patch": "c63e2a1d76b7a79dc2fb6431d9ddcf759abbe55c4b4206126fd8dc04872268d1",
```

Comment:
not sure patch files need to be in signatures file?

Reply:
Fixed.